.travis.yml

```diff
@@ -65,3 +65,14 @@ script:
   - if [ -n "$CLANG_FORMAT_PATH" ]; then bin/clangfmt --test; fi
   - bin/scalafmt --test
   - bash sbt -J-Xmx2G rebuild "set scriptedBufferLog in sbtScalaNative := false" test-all
+
+before_cache:
+  - find $HOME/.sbt -name "*.lock" | xargs rm
+  - find $HOME/.ivy2 -name "ivydata-*.properties" | xargs rm
+
+cache:
+  apt: true
```

Comment:
It looks like Travis only supported APT caching of private repositories (https://github.com/travis-ci/travis-ci/issues/5876 and https://github.com/travis-ci/docs-travis-ci-com/commit/bd8b56d501dbb47f76d139831c99f55ac69d1314).

Would you accept this PR if I rephrase the message to not fix #757 and remove this line?